meshtastic/tests/test_node.py

```
@@ -94,6 +128,25 @@ def _fake_send_admin(
     return _fake_send_admin
 
 
+def _get_mock_call_arg(call: Any, *, name: str, positional_index: int) -> Any:
```

Ruff (0.15.5)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `call`

(ANN401)